compiler-core/src/parse.rs

```diff
@@ -1289,6 +1289,23 @@ where
                             }
                         }
 
+                        Some((_, Token::Name { name: label }, int_e)) => {
```

Comment:
This doesn't support `thing.one.two`, which I think is a requirement. We'll need to parse more than a name